profiling/relecture-reference.md

```diff
@@ -0,0 +1,38 @@
+# Semantic
+
+## Bold
+- module python
+- service externe
+- language
+
+## Code
+- variable
+- fonction
+- méthode
+- paramètres
+- class
+- maths (hors $.*$)
+- caractère
+
```

Comment:
Je valide quand j'ai vu le rendu final, car je ne comprends pas les for/if/else en italique